src/store/media.js

```diff
@@ -15,10 +15,13 @@ import {
   RESET_MEDIA,
   SET_MEDIA_ITEM,
   SET_MEDIA,
+  RESET_FETCH_STATE,
 } from '~/constants/mutation-types'
 import { AUDIO, IMAGE, ALL_MEDIA, supportedMediaTypes } from '~/constants/media'
 import MediaService from '~/data/media-service'
 
+import { hash, rand as prng } from '~/utils/prng.ts'
```

Comment:
We shouldn't use the extension here. A lint rule to enforce this would be nice.

Reply:
The `import` linter plugin has a rule for this, we just need to configure it. I'll open an issue and have a PR for it soon :+1: